src/FixedDataTableBufferedRows.react.js

```diff
@@ -117,12 +117,22 @@ var FixedDataTableBufferedRows = React.createClass({
     var rowPositionGetter = props.rowPositionGetter;
 
     var rowsToRender = this.state.rowsToRender;
+
+    //Sort the rows, we slice first to avoid changing original
+    var sortedRowsToRender = rowsToRender.slice().sort((a, b) => a - b);
+    var rowPositions = {};
+
+    //Row position calculation requires that rows are calculated in order
+    sortedRowsToRender.forEach((rowIndex) => {
+      rowPositions[rowIndex] = rowPositionGetter(rowIndex);
```

Comment:
Does this still suffers the n*log(n) issue you mention here, right?
https://github.com/facebook/fixed-data-table/issues/405

Or is it only m*log(n) where m is the number of rows we render?

For the long term could we optimize the sumUntil method used for this case since I think we should be able to do it in m + log(n) time for this case?